### ISSUE TITLE
Change z score calculation sign and simulated_values name

### ISSUE THREAD

**Issue by hyunjimoon** [hyunjimoon/SBC#27]
1. Could we add some indication (prior or post) to`simulated_values` name? I was confused for a long time :D
It seems likely that `simulated_value` are prior sample values from [this](https://github.com/hyunjimoon/SBC/blob/cc189fafd8bbb1a36797f4fb9b9af61f255ab458/R/results.R#L430) code. 

2. Z score is calculated as follows.
![image](https://user-images.githubusercontent.com/30194633/129658119-038d35f1-726b-4c3f-883b-1037692f49b8.png)
![image](https://user-images.githubusercontent.com/30194633/129658137-11519e35-d5b3-47bf-8fc6-cd654994dde3.png)
Not serious, but unifying the sign of z-score with the definition might be beneficial. For rank calculation, it uses `sum(posterior < prior)` so maybe it is easier to remember it is posterior to prior order? 

`stats$z_score <- (stats$simulated_value - stats$mean) / stats$sd` -> `stats$z_score <- (stats$mean - stats$simulated_value ) / stats$sd` 

Check is needed @martinmodrak . Thanks!

```
> results$stats$simulated_value
 [1]  0.1214799  1.1831438 -0.6877552  0.0876214 -0.1980462  1.5717982 -2.7711167  0.0353704 -1.0338861  1.0901678
> SBC_fit_to_draws_matrix(results$fits[[1]])
# A draws_matrix: 1000 iterations, 4 chains, and 2 variables
    variable
draw lp__  theta
  1  -2.9 -0.201
  2  -2.9 -0.123
  3  -3.1 -0.377
  4  -3.4  0.165
  5  -3.3  0.117
  6  -3.5  0.208
  7  -2.9 -0.111
  8  -2.9 -0.260
  9  -2.9 -0.260
  10 -3.1  0.031
# ... with 3990 more draws
> SBC_fit_to_draws_matrix(results$fits[[2]])
# A draws_matrix: 1000 iterations, 4 chains, and 2 variables
    variable
draw lp__ theta
  1  -1.5   1.3
  2  -1.4   1.1
  3  -1.4   1.2
  4  -1.6   1.3
  5  -1.9   1.5
  6  -1.8   1.4
  7  -1.6   1.4
  8  -1.4   1.1
  9  -1.5   1.1
  10 -1.7   1.4
# ... with 3990 more draws
> SBC_fit_to_draws_matrix(results$fits[[3]])
# A draws_matrix: 1000 iterations, 4 chains, and 2 variables
    variable
draw lp__ theta
  1  -5.6 -0.63
  2  -5.9 -1.03
  3  -6.3 -1.15
```

**Comment by martinmodrak** [comment]
Good point about the naming! `prior_simulated_value` would be probably the most descriptive, but gets a bit long... (but is still acceptable). The man issue is that long variable names mean you get less columns when inspecting the `stats` data frame. Maybe `true_value` would also work? But not completely sure... maybe better to have a long name?

The sign of z-score is kind of arbitrary - I think Mike is trying to have "positive z-score means the posterior overestimates the parameter" while I was aiming for "positive z-score means the true parameter is larger than what the model thinks". The wikipedia definition (https://en.wikipedia.org/wiki/Standard_score) has the mean with a negative sign (as I do), but I have no strong feelings about this. In any case, the user would usually see both the "true" value and the posterior mean along with the z-score, so can check for themselves whether it is over- or under-estimation and the most important aspect of z-score is its magnitude... Maybe you pick the sign?

**Comment by martinmodrak** [comment]
This never got resolved, but I don't see a big value in changing things at this point - do you still feel like a change would be good?
